lib/event_factory.rb

```diff
@@ -89,4 +89,16 @@ def self.request_update_note_to_manager(request, user, message) # rubocop:todo M
     ).deliver_now
   end
   # rubocop:enable Metrics/MethodLength
+
+  # Creates an event for retention instructions when labware is updated
+  def self.record_retention_instruction_updates(labware, user, old_retention_instruction)
+    old_retention_instruction = 'nil' if [nil, ''].include?(old_retention_instruction)
```

Comment:
Wondering if you could use `blank?` or `empty?` instead of `[nil, ''].include?(old_retention_instruction)` 

https://blog.arkency.com/2017/07/nil-empty-blank-ruby-rails-difference/